<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupemodule</class>
 <widget class="QDialog" name="groupemodule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="matiere_tableView">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>10</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTableView" name="gm_tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>651</width>
     <height>201</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>CoefGM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="coef_doubleSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>162</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NomGM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_pushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="matiere_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Matiere</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idMat_lineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="LM_tableView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>256</width>
     <height>201</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>165</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="IdGM_label">
      <property name="text">
       <string>IdGM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IdGM_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="delete_pushButton">
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modify_pushButton">
      <property name="text">
       <string>update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_pushButton">
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
